.env-config

```diff
@@ -0,0 +1,8 @@
+# where the config file is
```

Comment:
This is my original(tm) idea. The goal is to be able to quickly set up a running oSparc-Full-Ops-Deployment using (as much as possible) only the `repo.config` file from the osparc-ops-repo. To this end I wrote a python script (sorta envsubst on steroids) that subsitutes the variables from  `repo.config` into `.env-template` for the deployment agent. More clearly: I have a script that generates an appropriate `.env-config` file from `.env-template` and `repo.config`. Then the makefile in the deployment agent repo in a second step uses `.env-config` and generates `.env` from it. In this step, any implicit references (such as `AAA=$(BBB)`) are resolved.

I dont use .env-devel currently. I couldnt really deduce the purpose from the name, except storing devel config? Not sure. For the procedure, you explained the name `.env-config`.

the git-commited `.env-config` is smaller because it is not-functioning :D This file shouldnt be in the git repo in the first place, as it should be generated (by hand, or using my python script, anyway at some point explicit machine config has to be stated) using .env-template. Its presence might be necessary for some unit/sys/int tests? Not sure at this point, if you want to I can follow up and clean this up properly asap.

We can change this if you have a better idea or if another approach is more in line with the #common way for z43/oSparc.

Reply:
Update: I have removed one abstraction layer today. Now, the deployment agent expects a valid `.env` file for the config. `.env-template` acts as a scaffold. There is no more `.env-config`. I rethought it and too many abstractions will only ause headaches, we can re-add this step incase it becomes truly necessary